inspirehep/utils/url.py

```diff
@@ -89,3 +90,17 @@ def retrieve_uri(uri, outdir=None):
 
         local_file.flush()
         yield local_file.name
+
+
+def get_prod_url_for_recid(recid, pattern_config_var='LEGACY_RECORD_URL_PATTERN'):
```

Comment:
We have to write code that is as environment-agnostic as possible. So I don't like the work `prod` in here. The word `LEGACY` is fine here instead as we can not run away from that.